src/kvirc/sparser/KviIrcServerParser_literalHandlers.cpp

```diff
@@ -954,9 +954,13 @@ void KviIrcServerParser::parseLiteralPrivmsg(KviIrcMessage * msg)
 		//			}
 		//		}
 
+		// "znc.in/self-message" capability: Handle a replayed message from ourselves to someone else.
+		bool bSelfMessage = IS_ME(msg, szNick);
+		QString szWindow = bSelfMessage ? szTarget : szNick;
```

Comment:
Oh, OK. Well, in that case, should I even bother updating this PR if no software exists that can generate such a message? (I'm on vacation and the soonest I can do that is over a week from now)